### ISSUE TITLE
User is not getting notified that a newly installed addon has been disabled needs to be enabled and configured manually

### ISSUE THREAD

**Issue by mYnDstrEAm** [xbmc/xbmc#18739]
### Describe the bug
Here is a clear and concise description of what the problem is:

When installing an addon like kodi-visualization-spectrum via a package-manager the addon is disabled upon starting Kodi and the user is not informed or prompted that s/he needs to a) enable the addon and/or b) configure it or any related settings for being able to use it. No pop-up or anything similar is shown.

* After installation I looked for "Spectrum" in Kodi's Addons tab and did not find it.
* I then searched for "Spectrum" in the search in the Addons tab and no search result was shown.
* I was told to look for the addon under User addons -> All and found it there and could enable it
* It still didn't show up under addons and the search results
* I [was told](url) to enable Spectrum under Settings -> Player -> Music -> Visualization
* When playing music tracks it now shows a visualization

This is very unintuitive and a large barrier to a user/newcomer-friendly Kodi that's also usable by less tech-savvy people and competitive to other large media centers.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

I expected to find the addon (or something like "Spectrum-Control") under the Addons tab, to find it in the search results and to be informed, for example via a clickable popup, that the addon needs to be enabled and configured.

## Actual Behavior

After starting Kodi no notification was shown; nothing indicated any change or required changes to make the newly installed addon usable.

## Possible Fix

* Making sure all installed addons show up in the Addon search results
* Showing all installed Addons or a button to show a list of all installed addons directly on the Addons page
* When a newly installed addon is detected (or installed) instead of disabling it, display a prompt that informs the user that the new addons needs to enabled, where the addon can be enabled and gives the options to enable or leave it disabled
* As an alternative at least a notification popup could be shown (it would be best if that was clickable so that it shows at least more information about how to enable the addon)
* If the addon requires changes in the settings prompt the user the user to make these changes, including a way to jump directly to these settings or to change the settings directly from within the prompt
* As an alternative or additionally maybe the setting to enable Spectrum under Settings -> Player -> Music -> Visualization could be made from the addon's configuration
* As an alternative or additionally the addon could be installed under to enable Spectrum under Settings -> Player -> Music -> Visualization (e.g. a "More options" that then shows all visualization addons available in the user's repositories)

### To Reproduce

1. Install kodi-visualization-spectrum via a package-manager 
2. Start Kodi
3. Try to show visualizations for music in the way described above including searching for the addon in the addons tab

### Debuglog

The debuglog can be found here:
https://pastebin.com/iJWvymJ3

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

[Here](https://forum.kodi.tv/showthread.php?tid=327763&pid=2701238#pid2701238) another user had a related problem.

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [x] Linux
 - [ ] OSX
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Debian 10 stable
 - Kodi version: Kodi 19.0

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by basilgello** [comment]
@ronie @howie-f makes sense probably to notify end-user about newly-installed addons requiring configuration just like versoncheck pops up?

**Comment by ronie** [comment]
> at least the addon not showing up in the search results seem to be a bug

i can confirm. perhaps @howie-f has an idea why binary addons are not included in the search results.
(the only binary addon i have installed is the PVR Demo Client, and it does not show when searching for it)

as for your other suggestions (how to make kodi more intuitive), the [feature requests](https://forum.kodi.tv/forumdisplay.php?fid=9) section on our forum would be the most suitable place to discus those.

**Comment by howie-f** [comment]
we do not see the addon in the search results because at least in case of linux they're installed via package manager or manually and not served from the official repo. search results actually reflect the repository content (and not the installed add-ons) so the results differ on windows for example where `visualization.*` and `pvr.*` are found.

agreed this is suboptimal but the way it is at the moment

**Comment by basilgello** [comment]
Right now, all newly installed addons via apt-get is prompted on the first Kodi following the installation. 19.x branch is affected, but 20.x and newer should not be.
